packaging/python/build_wheels.bash

```diff
@@ -151,31 +163,31 @@ case "$1" in
     USE_STATIC_READLINE=1
     python_wheel_version=${python_wheel_version//[-._]/}
     for py_bin in /opt/python/cp${python_wheel_version}*/bin/python; do
-        build_wheel_linux "$py_bin" "$bare" "$MPI_INCLUDE_HEADERS"
+        build_wheel_linux "$py_bin" "$coreneuron" "$MPI_INCLUDE_HEADERS"
     done
     ;;
 
   osx)
     MPI_INCLUDE_HEADERS="/usr/local/opt/openmpi/include;/usr/local/opt/mpich/include"
     USE_STATIC_READLINE=1
     for py_bin in /Library/Frameworks/Python.framework/Versions/${python_wheel_version}*/bin/python3; do
-        build_wheel_osx "$py_bin" "$bare" "$MPI_INCLUDE_HEADERS"
+        build_wheel_osx "$py_bin" "$coreneuron" "$MPI_INCLUDE_HEADERS"
     done
     ;;
 
   CI)
     if [ "$CI_OS_NAME" == "osx" ]; then
         MPI_INCLUDE_HEADERS="/usr/local/opt/openmpi/include;/usr/local/opt/mpich/include"
-        build_wheel_osx $(which python3) "$bare" "$MPI_INCLUDE_HEADERS"
+        build_wheel_osx $(which python3) "$coreneuron" "$MPI_INCLUDE_HEADERS"
     else
         MPI_INCLUDE_HEADERS="/usr/lib/x86_64-linux-gnu/openmpi/include;/usr/include/mpich"
-        build_wheel_linux $(which python3) "$bare" "$MPI_INCLUDE_HEADERS"
+        build_wheel_linux $(which python3) "$coreneuron" "$MPI_INCLUDE_HEADERS"
     fi
     ls wheelhouse/
     ;;
 
   *)
-    echo "Usage: $(basename $0) < linux | osx > [--bare]"
+    echo "Usage: $(basename $0) < linux | osx > [python version 36|37|38|3*]  [coreneuron | coreneuron-gpu]"
```

Comment:
```suggestion
    echo "Usage: $(basename $0) < linux | osx > [python version 36|37|38|39|3*]  [coreneuron | coreneuron-gpu]"
```
nitpick level: humongous.

Reply:
😄 By the way, you will be surprise to see Python 3.10 there:
```
# ls /opt/python/
cp310-cp310/      cp36-cp36m/       cp37-cp37m/       cp38-cp38/        cp39-cp39/        pp37-pypy37_pp73/
```